src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolver.vsdconfigxml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+
+     Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+-->
+<Configuration xmlns="http://schemas.microsoft.com/vstudio/vsdconfig/2008">
+  <DefineGuid Name="CSharpFunctionResolverId" Value="CFE62047-C343-4DE4-B31E-88B7AFF8E7F7"/>
+  <ManagedComponent
+    ComponentId="CSharpFunctionResolverId"
+    ComponentLevel="65100"
+    Synchronized="true"
+    AssemblyName="Microsoft.CodeAnalysis.ExpressionEvaluator.FunctionResolver">
+    <Class Name="Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.CSharpFunctionResolver">
+      <Implements>
+        <InterfaceGroup Priority="Normal">
+          <NoFilter/>
+          <Interface Name="IDkmRuntimeFunctionResolver"/>
+        </InterfaceGroup>
+        <InterfaceGroup CallOnlyWhenLoaded="true" Priority="Normal">
+          <NoFilter/>
```

Comment:
Apparently my comment didn't show up...
You'll want to add this filter for OnModuleInstanceLoadNotification:
```
<Filter>
  <RuntimeId RequiredValue="DkmRuntimeId.Clr"/>
</Filter>
```